ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverBatch.java

```diff
@@ -49,27 +46,20 @@ void upsertKeyspace(Session session) {
 
     @Override
     void upsertCF(Session session) {
-        session.execute("CREATE TABLE IF NOT EXISTS "+TableName+" (cyclist_name text, balance float STATIC, expense_id int, amount float, description text, paid boolean, PRIMARY KEY (cyclist_name, expense_id) )");
+        session.execute("CREATE TABLE IF NOT EXISTS " + tableName + " (cyclist_name text, balance float STATIC, expense_id int, amount float, description text, paid boolean, PRIMARY KEY (cyclist_name, expense_id) )");
         session.execute("CREATE TABLE IF NOT EXISTS "+TableName2+" (expense_id int, cyclist_name text, PRIMARY KEY (expense_id, cyclist_name)) ");
     }
 
     @Override
     void preInit() {
         this.TableName2 = config.getCfname2();
-        this.batchSize = config.getBatchSize();
-        this.useTimeStamp = config.getUseTimestamp();
-        this.useMultiPartition = config.getUseMultiPartition();
-        this.TableName = config.getCfname();
-        this.ReadConsistencyLevel = ConsistencyLevel.valueOf(config.getReadConsistencyLevel());
-        this.WriteConsistencyLevel = ConsistencyLevel.valueOf(config.getWriteConsistencyLevel());
-        this.MaxColCount = config.getColsPerRow();
     }
 
     @Override
     public String readSingle(String key) throws Exception {
         BoundStatement statement = readPstmt.bind();
         statement.setString("cyclist_name", key);
-        statement.setConsistencyLevel(this.ReadConsistencyLevel);
+        statement.setConsistencyLevel(ConsistencyLevel.valueOf(config.getReadConsistencyLevel()));
```

Comment:
What happens if somebody provides an incorrect consistency level in the configuration? Shall we have an exception here or not?